src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,32 @@
-export const TodoInfo = () => {};
+import classNames from 'classnames';
+import { Todo } from '../../App';
+import { UserInfo } from '../UserInfo';
+
+interface UserTodo {
+  userId: number;
+  userName: string;
+  userEmail: string;
+  userLogin: string;
+  todoPost: Pick<Todo, 'completed' | 'id' | 'title'>;
+}
+
+type Props = {
+  todo: Pick<UserTodo, 'userName' | 'userEmail' | 'todoPost'>;
+};
```

Comment:
It doesn't make much sense to create another one interface and then just pick a few properties from it